__tests__/app.test.js

```diff
@@ -30,16 +31,47 @@ describe('GET/api/topics', () => {
                 expect(topic).toHaveProperty('description')
             })
         })
-        
+    })
+})
+describe('GET/api', () => {
+    test('200: should respond with an object describing all available endpoints', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            const resEndpoints = body
+            expect(resEndpoints).toEqual(endpoints)
+        })
+    })
+    test('200: should respond with a nested object, with each inner object given an appropriate name', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            const endpointNames = Object.keys(body)
+            const regex = /(^GET |^POST |^PATCH |^DELETE )(\/api)/
+            endpointNames.forEach((endpointName) => {
+                expect(regex.test(endpointName)).toBe(true)
+            })
+        })
+    })
+    test('200: should respond with objects containing the correct properties', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            const endpointsDetails = Object.values(body)
+            endpointsDetails.forEach((endpointDetails) => {
+                expect(typeof endpointDetails.description).toEqual('string')
+                expect(typeof endpointDetails.queries).toEqual('object')
+                expect(typeof endpointDetails.exampleResponse).toEqual('object')
```

Comment:
These two tests aren't adding anything new that isn't already being covered by your first test. The first test is using `toEqual` which ensure that all the property names and values are all matching, so it's all being handled nice and concisely already, these two tests could be removed to save lines of code